tests/test_web_dashboard_reboot.py

```
@@ -184,3 +191,144 @@ def test_probe_droids_degraded_when_bridge_missing(monkeypatch, tmp_path):
+                "id": "action-1",
+                "action_id": "compile_dashboard",
+                "label": "Compile Dashboard",
+                "status": "failed",
+                "exit_code": 1,
+                "duration_ms": 10,
+                "started_at": "2026-04-28T00:00:01Z",
+                "completed_at": "2026-04-28T00:00:02Z",
+                "stderr_preview": "password=[redacted]",
+                "stdout_preview": "",
+                "command": ["python3"],
+                "cwd": "/tmp",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)